### DISCUSSION TITLE
How to remove rate limiter for forgot-password?

### DISCUSSION THREAD

**Discussion by amlxv** [laravel/framework#48273] [Q&A] [answered]
## Laravel Fortify. 

It is for development purpose. I keep getting "Please wait before retrying.". 

FortifyServiceProvider.php

```php
RateLimiter::for('forgot-password', function (Request $request) {
    return Limit::none();
});
```

fortify.php

```php
'limiters' => [
    'login' => 'login',
    'two-factor' => 'two-factor',
    'forgot-password' => 'forgot-password'
],
```

**Comment by henzeb** [comment]
Came across this post. The question was for laravel 8, but it may work in 10 as well. https://stackoverflow.com/questions/66485790/how-to-ratelimit-forgot-password-requests-in-laravel-fortify

**Comment by amlxv** [comment, marked as answer]
Thank you @henzeb for your response. I've found the solution.

config/auth.php

```php
'passwords' => [
    'users' => [    
        'provider' => 'users',
        'table' => 'password_reset_tokens',
        'expire' => 60,
        'throttle' => 0, // default: 60
    ],
],
```
